### DISCUSSION TITLE
Setting the number of cores ?

### DISCUSSION THREAD

**Discussion by Elimelech** [pgj/freebsd-wifibox#55] [Q&A] [answered]
Where do I need to change the settings to set the number of cores I need if I have a multi-core processor?
So
/usr/local/etc/wifibox/bhyve.conf
 or like this:
in /etc/sysctl.conf
```
echo "hw.vmm.topology.cores_per_set=8" >> /etc/sysctl.conf
echo "hw.vmm.topology.cpusets=1" >> /etc/sysctl.conf
```

**Comment by pgj** [comment, marked as answer]
That is `bhyve.conf`.  You can also see a comment about that in the sample file:

```shell
cpus=1          # number of CPUs allocated for the guest
```

This option translates to the `-c` flag of `bhyve(8)`.
